lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -1401,6 +1408,58 @@ static void maybeDiagnoseClassWithoutInitializers(ClassDecl *classDecl) {
   diagnoseClassWithoutInitializers(classDecl);
 }
 
+//void TypeChecker::checkParameterList(ParameterList *params,
```

Comment:
What's going on here?

Reply:
Whoops sorry going to clean this up right away... I had two "places" I was trying to do the typechecking for the parameters, it is now centralized in checkDistributedFunction in `TypeCheckDistributed.cpp` removing those new (and commented) funcs